config/paths.js

```diff
@@ -1,5 +1,7 @@
 const { dirname, join } = require('path')
 
+const pkg = require('../package/package.json')
```

Comment:
Nice trick to not `readFile` and `JSON.parse` and all 🙌🏻

Reply:
Moved it here:

* https://github.com/alphagov/govuk-frontend/pull/3113